website/content/partials/deprecation/duplicate-hcl-attributes.mdx

```diff
@@ -4,7 +4,8 @@
 | :-------: | :---------------------: | :--------------: |
 | JUN 2025  | OCT 2025                | N/A
 
-The ability to duplicate attributes in HCL configuration files and policy definitions is deprecated.
+In Vault v1.20.x and v1.19.6, the ability to duplicate attributes in HCL configuration
+files and policy definitions is deprecated.
```

Comment:
In Vault v1.20.x and v1.19.6+, the ability to duplicate attributes in HCL configuration
files and policy definitions is deprecated.